### ISSUE TITLE
TV Input Framework Companion Library depends on support libraries 24.0.0

### ISSUE THREAD

**Issue by PiotrWozniakSC** [googlesamples/androidtv-sample-inputs#22]
The latest version of TV Input Framework Companion Library (0.2) depends specifically on support libraries from SDK 24 (as seen when `./gradlew -q dependencies app:dependencies --configuration compile` is run):
`+--- com.google.android.libraries.tv:companionlibrary:0.2
|    +--- com.android.support:appcompat-v7:24.0.0
|    |    +--- com.android.support:support-vector-drawable:24.0.0
|    |    \--- com.android.support:animated-vector-drawable:24.0.0
|    |         \--- com.android.support:support-vector-drawable:24.0.0`

However, SDK 26 is already available, and using it alongside Companion Library 0.2 causes warnings in gradle about mixing versions. We haven't experienced any issues as of yet, but, as the warning rightly suggests, mixing versions can lead to runtime crashes.

My question is: will a new version of Companion Library supporting SDK 26 be released? And if so, when can such a release be expected?

Additionally, I noticed that README.md on https://github.com/googlesamples/androidtv-sample-inputs is outdated - it still suggests using version 0.1 in the "TIF Companion Library" section.

**Comment by falxcerebri** [comment]
I've just bumped into this issue while trying to integrate Firebase into ATV project. ARGH...

**Comment by Fleker** [comment]
Yeah I have noticed this as well. In the short-term you may be able to get away with using this in your build.gradle:

```
implementation('com.google.android.libraries.tv:companionlibrary:0.2') {
        exclude group: "com.google.ads.interactivemedia.v3"
        exclude group: "com.google.android.gms"
        exclude group: "com.android.support"
    }
```
